labgrid/driver/ubootdriver.py

```diff
@@ -95,17 +95,21 @@ def _run(self, cmd: str, *, timeout: int = 30, codec: str = "utf-8", decodeerror
 
     @Driver.check_active
     @step(args=['cmd'], result=True)
-    def run(self, cmd, timeout=None): # pylint: disable=unused-argument
+    def run(self, cmd, timeout=None):
```

Comment:
This should be changed to `def run(self, cmd, timeout=30)` to match the other drivers.

Reply:
Alright, sounds reasonable.